endpoints/openrtb2/auction_test.go

```diff
@@ -137,41 +148,107 @@ func TestImplicitUserId(t *testing.T) {
 
 // TestGoodRequests makes sure we return 200s on good requests.
 func TestGoodRequests(t *testing.T) {
-	assertResponseFromDirectory(t, "sample-requests/valid-whole/exemplary", getRequestPayload, nilReturner, http.StatusOK, true)
-	assertResponseFromDirectory(t, "sample-requests/valid-whole/supplementary", noop, nilReturner, http.StatusOK, true)
+	exemplary := &getResponseFromDirectory{
```

Comment:
good change!

At some point it might be worth experimenting with testify's Suite to see it if would make things here nicer: https://godoc.org/github.com/stretchr/testify/suite

but... not this PR.